src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -108,9 +108,11 @@ namespace SurfaceGeometry {
 
     static std::string const BlankString;
 
-    int constexpr UnenteredAdjacentZoneSurface = -998; // allows users to enter one zone surface ("Zone")
-                                                       // referencing another in adjacent zone
-    int constexpr UnreconciledZoneSurface = -999;      // interim value between entering surfaces ("Surface") and reconciling
+    int constexpr unenteredAdjacentSpaceSurface = -997; // allows users to enter one zone surface ("Space")
+                                                        // referencing another in adjacent space
+    int constexpr unenteredAdjacentZoneSurface = -998;  // allows users to enter one zone surface ("Zone")
+                                                        // referencing another in adjacent zone
+    int constexpr unreconciledZoneSurface = -999;       // interim value between entering surfaces ("Surface") and reconciling
```

Comment:
Somewhat gratuitous name changes to lower case to help reveal where these flags get used.